security/migrations/0001_migration.py

```diff
@@ -25,9 +24,9 @@ class Migration(migrations.Migration):
                 ('queue_name', models.CharField(blank=True, max_length=250, null=True, verbose_name='queue name')),
                 ('input', models.TextField(blank=True, editable=False, null=True, verbose_name='input')),
                 ('task_args',
-                 jsonfield.fields.JSONField(blank=True, editable=False, null=True, verbose_name='task args')),
+                 models.TextField(blank=True, editable=False, null=True, verbose_name='task args')),
```

Comment:
Why not JSONField?

Reply:
because there is changed old migration. jsonfield.fields.JSONField extends TextField. So If I want remove jsonfield from libraries I must use TextField. Next migration will change it to the JSONField with the db type migration